lenra-io/server
# feat: Manage not authenticated users

## About this PR

Closes #539 

## Technical highlight/advice
I made a channel test case that made me create many files...

## How to test my changes
- Create an app with routes available for guest users (see https://github.com/lenra-io/api/issues/15)
- Start it (with JSON view since Lenra view need a client adaptation)

## Checklist

### I included unit tests that cover my changes

### I added/updated the documentation about my changes
